Resources/public/js/views/fields/ez-float-editview.js

```diff
@@ -13,7 +13,7 @@ YUI.add('ez-float-editview', function (Y) {
 
     var L = Y.Lang,
         FIELDTYPE_IDENTIFIER = 'ezfloat',
-        FLOAT_PATTERN = "\\-?\\d*\\.?\\d+"; // WARNING: each backslash is doubled, because it is escaped on output otherwise
```

Comment:
Yes. The purpose of the patch was to not only fix the firefox (mis)behavior, but to provide better cross-browser compatibility¹ and making sure validation does work.

\[1\]: http://caniuse.com/#feat=input-pattern